src/main/java/org/reaktivity/nukleus/oauth/internal/OAuthRealms.java

```diff
@@ -97,7 +161,7 @@ public boolean unresolve(
         }
         else
         {
-            result = realmsIdsByName.entrySet().removeIf(e -> (e.getValue() == scope));
+            result = realmsIdsByName.entrySet().removeIf(e -> (e.getValue().realmBit == scope));
```

Comment:
Actually, I was confused by `SCOPE_MASK`.
Please rename `SCOPE_MASK` to `REALM_MASK` since it will retain only the realm after the mask is applied.
Then the result of `authorization & REALM_MASK` is really `realmId`, not `scopeBits`.